### PR TITLE
Fixes quick equip not respecting access delays

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Fixes quick equip bypassing access delays, and bypassing the draw restriction on dispenser.
## Why It's Good For The Game
Bug fix good
## Changelog
:cl:
fix: fixed quick equip bypassing access delays and dispenser restrictions
/:cl:

### CODE REVIEW

**Comment by Lumipharon** [comment]
I've done a fug, hold on.

**Comment by Derrick2Electricboogaloo** [comment]
> I've done a fug, hold on.

Do an unfug

**Comment by Lumipharon** [comment]
> > I've done a fug, hold on.
> 
> Do an unfug

It unfugged

File: code/game/objects/items/storage/dispenser.dm [hunk 1]
```diff
@@ -132,3 +132,6 @@
 
 /obj/item/storage/backpack/dispenser/attempt_draw_object(mob/living/user)
 	to_chat(usr, span_notice("You can't grab anything out of [src] while its not deployed."))
+
+/obj/item/storage/backpack/dispenser/do_quick_equip(mob/user)
+	to_chat(usr, span_notice("You can't grab anything out of [src] while its not deployed."))
```
Comment (QualityVan) [inline code/game/objects/items/storage/dispenser.dm:137]:
```suggestion
	to_chat(usr, span_notice("You can't grab anything out of [src] while it's not deployed."))

/obj/item/storage/backpack/dispenser/do_quick_equip(mob/user)
	to_chat(usr, span_notice("You can't grab anything out of [src] while it's not deployed."))
```

**Review by BraveMole — APPROVED** [review]

**Pull request merged**